### DISCUSSION TITLE
程式碼和導航

### DISCUSSION THREAD

**Discussion by Gu-myhome** [community/community#133792] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

這樣子看起來是沒什麼問題耶對啊，可能是我經驗不夠吧，然後還有很多地方要學😳

**Comment by grandmas-favorite** [comment]
Hi @Gu-myhome , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Reply by Gu-myhome** [reply]
這是導航嗎？
